src/chrome/content/rules/Frugalware.org.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+	Invalid certficate:
+		darcs.frugalware.org
+		dors.frugalware.org
+		git.frugalware.org
+		lists.frugalware.org
+		packages.frugalware.org
+		planet.frugalware.org
+		wiki.frugalware.org
+
+	Private subdomain:
+		webmail.frugalware.org
```

Comment:
As long as we can get a HTTP 200 response we should add it anyway. Maybe some people there are using HTTPS Everywhere too ;)